hilti/include/ast/ctors/bool.h

```diff
@@ -4,14 +4,15 @@
 
 #include <hilti/ast/ctor.h>
 #include <hilti/ast/types/bool.h>
+#include <hilti/rt/types/bool.h>
 
 namespace hilti {
 namespace ctor {
 
 /** AST node for a boolean constructor. */
 class Bool : public NodeBase, public hilti::trait::isCtor {
 public:
-    Bool(bool v, Meta m = Meta()) : NodeBase(std::move(m)), _value(v) {}
+    Bool(hilti::rt::Bool v, Meta m = Meta()) : NodeBase(std::move(m)), _value(v) {}
```

Comment:
I think I'd leave this as `bool` for the AST side. Don't much of an advantage of switching to `Bool` there, but reduces readability.